Src/Core/Common/ExpressionExtensions.cs

```diff
@@ -129,9 +131,17 @@ public static class ExpressionExtensions
                         node = null;
                         break;
 
+                    case ExpressionType.Call:
+                        var methodCallExpression = node as MethodCallExpression;
```

Comment:
Good point - I'd assumed it could be something else, but doesn't look like that's the case. Updated accordingly.